src/server/routers/data/index.ts

```diff
@@ -1,12 +1,18 @@
 import { baseProcedure } from '../../procedures';
-import { getDataDashboardProjectByLabelSchema } from '../../../lib/services/data/schemas';
-import { getDataDashboardProject } from '../../../lib/services/data';
+import { getDataDashboardProjectByIdSchema } from '../../../lib/services/data/schemas';
+import {
+  getDataDashboardProject,
+  getDataDashboardProjects,
+} from '../../../lib/services/data';
 
 import { t } from 'server/trpc';
 
 const dataRouter = t.router({
+  getDataDashboardProjects: baseProcedure.query(async () => {
+    return await getDataDashboardProjects();
+  }),
   getDataDashboardProject: baseProcedure
-    .input(getDataDashboardProjectByLabelSchema)
+    .input(getDataDashboardProjectByIdSchema)
     .query(async ({ input }) => {
       return await getDataDashboardProject(input);
```

Comment:
Is there any case where it would be valid to pass no project at all?
If not please remove the optional from the zod object and check for an input in here.

Reply:
A project should always be given as input. In the services prisma query I return an error 404 if the project is not found, do you want me to send a "Bad format" error before awaiting for the query if the input is undefined/null?

Reply:
Should be done